controller/array_action/array_action_types.py

```diff
@@ -1,15 +1,23 @@
-class Volume:
-    def __init__(self, vol_size_bytes, vol_id, vol_name, array_address, pool_name, copy_source_id, array_type):
+class TypeRepresent:
```

Comment:
since we're not implementing `__repr__`, i would use a different term.
maybe `LogFriendly`/`CustomStr`/`MembersPrintable`?

Reply:
done